src/components/NavBar.jsx

```diff
@@ -1,19 +1,29 @@
+import React from 'react';
+
 import {
   Link,
 } from 'react-router-dom';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faCircleUser } from '@fortawesome/free-solid-svg-icons';
+import './NavBar.css';
 
 const Navbar = () => (
-  <nav className="navbar-container">
-    <h1>Bookstore</h1>
+  <nav className="nav-bar">
+    <div className="nav-brand">
+      <Link to="/bookstore-cms/"><h1>Bookstore CMS</h1></Link>
+    </div>
     <ul className="nav-links">
       <li>
-        <Link to="/">Home</Link>
+        <Link className="nav-link" to="/bookstore-cms/">BOOKS</Link>
       </li>
+      <li>|</li>
       <li>
-        <Link to="/categories">categories</Link>
+        <Link className="nav-link" to="/bookstore-cms/categories">CATEGORIES</Link>
```

Comment:
Your design
![image](https://github.com/coker2019/booksave/assets/78251927/130f970f-43ea-42e1-8b31-a4e43d07068a)

Template
![image](https://github.com/coker2019/booksave/assets/78251927/cf3ec9ef-4c64-47eb-8196-4f739723b8a2)

Since our aim is to make our final design match [Zeplin's design](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378), I would suggest that we remove the `Line` in the `navbar` as indicated above, as it is not present in the template. what do you think? ☺